docs/scripts/translate-qa.py

```
@@ -0,0 +1,424 @@
+"""
+import os
+import sys
+import json
+import argparse
+import re
+from pathlib import Path
+
+try:
+    import requests
+except ImportError:
+    os.system("pip3 install requests -q")
```

Ruff (0.15.12)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)